tests/polarion_reporter.go

```diff
@@ -0,0 +1,128 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package tests
+
+import (
+	"encoding/xml"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/types"
+)
+
+var Polarion = PolarionReporter{}
+
+func init() {
+	flag.BoolVar(&Polarion.Run, "polarion", false, "Run polarion reporter")
+	flag.StringVar(&Polarion.projectId, "polarion-project-id", "", "Set the Polarion project ID")
+	flag.StringVar(&Polarion.filename, "polarion-report-file", "polarion.xml", "Set the polarion report file path")
```

Comment:
Polarion should be capitalized in the description string for consistency

Reply:
fixed